test/test_main.py

```diff
@@ -898,9 +908,10 @@ def test_hook_and_dispatch(self):
              'Using local storage: not a Kubernetes podspec charm'],
             ['juju-log', '--log-level', 'DEBUG', '--',
              'Emitting Juju event install.'],
+            ['is-leader', '--format=json'],
         ]
         calls = fake_script_calls(self)
-        self.assertRegex(' '.join(calls.pop(-2)), 'Initializing SQLite local storage: ')
+        self.assertRegex(' '.join(calls.pop(-3)), 'Initializing SQLite local storage: ')
```

Comment:
I do *not* like how we have these brittle indexes in the tests, but I've left that as is for future cleanup work!